CODE-GUIDELINES.md

```diff
@@ -37,3 +37,97 @@ test('...', () => {
   vi.mocked(window.windowMethod).mockResolvedValue('a string');
 });
 ```
+
+### Mock complete modules, spy on parts of module for specific tests
```

Comment:
Does it apply also to internal modules ?

Reply:
I would not like to enforce this point, it would be to the developer to choose the limit of what is mocked/unit tested. I'll develop a little bit more to clarify this point